include/openthread/netdiag.h

```diff
@@ -234,6 +252,15 @@ typedef struct otNetworkDiagChildEntry
     otLinkModeConfig mMode;    ///< Link mode.
 } otNetworkDiagChildEntry;
 
+/**
+ * Represents a Network Diagnostic Child Table TLV value.
+ */
+typedef struct otNetworkDiagChildTable
+{
+    uint8_t                 mCount; ///< Number of child entries in the table.
+    otNetworkDiagChildEntry mTable[OT_NETWORK_BASE_TLV_MAX_LENGTH / sizeof(otNetworkDiagChildEntry)]; ///< Child table.
```

Comment:
Did make pretty format the comments like this?

Reply:
Yes.  it can do weird stuff in some situation.